src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const semicolon = ';';
+  const colon = ':';
+
+  const convertedToObject = sourceString
```

Comment:
What does reduce return? Can u return it immediately?
```suggestion
  return sourceString
```